xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -437,6 +437,14 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
       case LISTITEM_VIDEO_ASPECT:
         value = CStreamDetails::VideoAspectToAspectDescription(tag->m_streamDetails.GetVideoAspect());
         return true;
+      case LISTITEM_VIDEO_WIDTH:
+        if (int val = tag->m_streamDetails.GetVideoWidth(); val > 0)
+          value = std::to_string(val);
+        return true;
+      case LISTITEM_VIDEO_HEIGHT:
+        if (int val = tag->m_streamDetails.GetVideoHeight(); val > 0)
```

Comment:
Same comments apply here as well

Reply:
:sigh:

Reply:
@graham8 no need to sigh; we're here to help you. Please understand that we try to ensure a certain level of code quality. All will be quicker with your next guiinfo label PR. ;-)